addons/common/initSettings.sqf

```diff
@@ -16,3 +16,12 @@
     true,
     true
 ] call CBA_settings_fnc_init;
+
+[
+    QGVAR(colorfulProgressBar),
+    "CHECKBOX",
+    [LSTRING(ColorfulProgressBar), LSTRING(ColorfulProgressBar_Description)],
+    localize LSTRING(ACEKeybindCategoryCommon),
+    false,
```

Comment:
Not sure, left it off by default since I thought most people would want keep the old look. Can change if you think it should be the default look.

Reply:
I would set it to true and if someone objects it can be turned off :)

Reply:
I even doubt if setting is necessary. @jonpas knows better anyway 😆

Reply:
I'd be tempted to say just make this the default, it might not work the same for all colourblind, but they would still be able to see the bar.

That said, aren't we deprecating progress bar for CBA implementation anyway?